pulp_python/app/models.py

```diff
@@ -1,35 +1,64 @@
-from gettext import gettext as _
 from logging import getLogger
 
 from django.db import models
 
-from pulpcore.plugin.models import (Artifact, Content, ContentArtifact, RemoteArtifact, Importer,
-                                    ProgressBar, Publisher, RepositoryContent, PublishedArtifact,
-                                    PublishedMetadata)
-from pulpcore.plugin.tasking import Task
-
+from pulpcore.plugin.models import Content, Importer, Publisher
+from django.contrib.postgres.fields import ArrayField
 
 log = getLogger(__name__)
 
 
-class PythonContent(Content):
+class Classifier(models.Model):
     """
-    The "python" content type.
+    Custom tags for classifier
+
+    Fields:
 
-    Define fields you need for your new content type and
-    specify uniqueness constraint to identify unit of this type.
+        name (models.TextField): The name of the classifier
 
-    For example::
+    Relations:
 
-        field1 = models.TextField()
-        field2 = models.IntegerField()
-        field3 = models.CharField()
+        pythonpackagecontent (models.ForeignKey): The task this tag is associated with
+
+    """
 
-        class Meta:
-            unique_together = (field1, field2)
+    name = models.TextField()
+    python_package_content = models.ForeignKey("PythonPackageContent", related_name="classifiers",
+                                               related_query_name="classifier")
+
+
+class PythonPackageContent(Content):
+    """
+    A Content Type representing Python's Distribution Package as
+    defined in pep-0426 and pep-0345
+    https://www.python.org/dev/peps/pep-0491/
+    https://www.python.org/dev/peps/pep-0345/
     """
 
     TYPE = 'python'
+    filename = models.TextField(unique=True, db_index=True, blank=False)
+    packagetype = models.TextField(blank=False)
+    name = models.TextField(blank=False)
+    version = models.TextField(blank=False)
+    metadata_version = models.TextField(blank=False)
+    summary = models.TextField()
+    description = models.TextField()
+    keywords = ArrayField(models.TextField())
+    home_page = models.URLField()
```

Comment:
Do these other URL fields have the same problem? Rather than worry about it, maybe just make them all text fields, and we can fiddle with this when we do validations.